Classes/Projection/ProjectionManager.php

```diff
@@ -88,18 +88,29 @@ public function getProjection(string $projectionIdentifier): Projection
     }
 
     /**
+     * Replay events of the specified projection
+     *
      * @param string $projectionIdentifier
-     * @return void
+     * @return integer Number of events which have been replayed
```

Comment:
Good call. I wanted to add some simple pub/sub to this (similar to the core migration manager) in order to be able to display progress on the CLI. But we can always replace this once done